.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+ 
+**General remarks**
+
+> Please delete this section including header before submitting
```

Comment:
You can comment out this block, so the text won't be displayed once is submited 
<!-- You can't see me -->
```
<!---
Please delete etc
-->
```

But i'm fine to merge as it, is better than nothing :)